src/explore-education-statistics-common/src/components/Pagination.tsx

```diff
@@ -0,0 +1,116 @@
+import { ArrowLeft, ArrowRight } from '@common/components/ArrowIcons';
+import generatePageNumbers from '@common/components/util/generatePageNumbers';
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+const paginationLinkClassName = 'govuk-link govuk-pagination__link';
+
+interface LinkRenderProps {
+  'aria-current'?: 'page' | undefined;
+  'aria-label'?: string;
+  children: ReactNode;
+  className: string;
+  rel?: 'next' | 'prev';
+  to: string;
+}
+
+export interface PaginationProps {
+  baseUrl: string;
```

Comment:
Would suggest making this optional and default it to an empty string. In these cases, the `to` render props would just be set to `?page=1` etc.